docs/type-declarations.md

```diff
@@ -1,20 +1,18 @@
 ---
-title: Writing Declarations
+title: Type Declarations
```

Comment:
Please dont rename this file, the name is fine and it's going to break all the links to this page out there